Backends/CLX/input.lisp

```diff
@@ -143,8 +143,10 @@
 	((:button-press :button-release)
 	 (let ((modifier-state (clim-xcommon:x-event-state-modifiers *clx-port* state))
                (button (decode-x-button-code code)))
-           (if (member button '(#.+pointer-wheel-up+ #.+pointer-wheel-down+
-                                #.+pointer-wheel-left+ #.+pointer-wheel-right+))
+           (if (and
+                (member button '(#.+pointer-wheel-up+ #.+pointer-wheel-down+
+                                 #.+pointer-wheel-left+ #.+pointer-wheel-right+))
+                (eq event-key :button-press))
                (make-instance 'climi::pointer-scroll-event
```

Comment:
Nice catch! thanks. Could you please change order of test and indent? `eq` should go first and it should be in the same line as `IF`. Also please put `#.+…` in separate lines.

```
(if (and (eq event-key …)
         (member '(#.+a+
                   #.+b+
                   #.+c+ …))
```

Reply:
thanks. no problem. Regarding PR's, if you force-push (or even simply push) to the branch PR is opened from, it will update itself automatically.